src/coord/src/coord.rs

```diff
@@ -257,6 +257,14 @@ pub struct Config {
     pub metrics_registry: MetricsRegistry,
     pub persister: PersisterWithConfig,
     pub now: NowFn,
+    pub librdkafka_debug: tracing::Level,
+}
+
+/// Extra context to pass through to connector creation
```

Comment:
Stale now?

Reply:
This is used in `coord::sink_connector`?